Projects/App/Sources/Screens/OnboardingScreen/WebOnboardingState.swift

```diff
@@ -0,0 +1,68 @@
+import Apollo
+import Flow
+import Foundation
+import hCore
+import hGraphQL
+import UIKit
+import WebKit
+
+enum WebOnboardingScreen {
+    case webOffer(ids: [String])
+    case webOnboarding
+}
+
+struct WebOnboardingState {
+    let screen: WebOnboardingScreen
+    
+    private var locale: String {
+        Localization.Locale.currentLocale.webPath
+    }
+    
+    private var path: String {
+        switch screen {
+        case .webOffer:
+            return "/\(locale)/new-member/offer"
+        case .webOnboarding:
+            return "\(locale)new-member"
+        }
+    }
+    
+    private var host: String {
+        Environment.current.baseUrl
+    }
+    
+    private var token: String? {
+        guard let token = ApolloClient.retreiveToken() else {
+            return nil
+        }
+        
+        return token.urlEncodedString
+    }
+    
+    private var queryItems: [URLQueryItem] {
+        switch screen {
+        case .webOffer(let ids):
+            return [URLQueryItem(name: "quoteIds", value: "[" + ids.joined(separator: ",") + "]")]
```

Comment:
What happens if we JSON encode `ids`? wouldn't that result in "[id1, id2]" maybe a bit cleaner?

Reply:
Idk why but it encodes the " when I do that... I thought the same.

Reply:
I guess cuz its [ "asd" : "asd" ] rather than "[asd,asd]"